app.py

```diff
@@ -38,7 +38,7 @@
     status_forcelist=[500, 502, 503, 504]
 )
 uncached_session.mount(
-    'https://api.snapcraft.io',
+    'https://api.stagin.snapcraft.io',
```

Comment:
Looks like a typo for `staging` here.

Reply:
this code should be rebased with the last version of s.io 
The URLs are handled in a different way now

Reply:
This is not the MR you're looking for